### PR TITLE
Create 2020-25-05-coreweekly-20-2020.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+
+## Code changes in modules, themes & tools
+
+
+### Changes in developer documentation
+* [#540](https://github.com/PrestaShop/docs/pull/540): Add a validation rule to avoid collisions between modules , by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#539](https://github.com/PrestaShop/docs/pull/539): Update dependencies for security reasons, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#536](https://github.com/PrestaShop/docs/pull/536): Add info about the CQRS debug toolbar, by [@jolelievre](https://github.com/jolelievre)
+* [#535](https://github.com/PrestaShop/docs/pull/535): Update _index.md. Thank you [@derekjwhitten](https://github.com/derekjwhitten)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md]:
```suggestion
* [#535](https://github.com/PrestaShop/docs/pull/535): Update index.md. Thank you [@derekjwhitten](https://github.com/derekjwhitten)
```

Comment (ttoine) [reply news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md]: will you edit the pull request title if I change it in the core weekly?

Comment (Progi1984) [reply news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md]: Euh no ! It's the file name ;) `_index.md`

Comment (ttoine) [reply news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md]: so why is there a _ in the pr title?

Comment (Progi1984) [reply news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md]: Because it is in the file path.

Comment (ttoine) [reply news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md]: I  still don't get it, but let's commit this and merge :-)

File: news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ This edition of the Core Weekly report highlights changes in PrestaShop's core c
 * [#540](https://github.com/PrestaShop/docs/pull/540): Add a validation rule to avoid collisions between modules , by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
 * [#539](https://github.com/PrestaShop/docs/pull/539): Update dependencies for security reasons, by [@PierreRambaud](https://github.com/PierreRambaud)
 * [#536](https://github.com/PrestaShop/docs/pull/536): Add info about the CQRS debug toolbar, by [@jolelievre](https://github.com/jolelievre)
-* [#535](https://github.com/PrestaShop/docs/pull/535): Update _index.md. Thank you [@derekjwhitten](https://github.com/derekjwhitten)
+* [#535](https://github.com/PrestaShop/docs/pull/535): Update index.md. Thank you [@derekjwhitten](https://github.com/derekjwhitten)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md:85]:
```suggestion
* [#535](https://github.com/PrestaShop/docs/pull/535): Update `_index.md`. Thank you [@derekjwhitten](https://github.com/derekjwhitten)
```

Comment (ttoine) [reply news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md:85]: ???

**Pull request merged**